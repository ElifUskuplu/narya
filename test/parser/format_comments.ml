open Testutil.Print

let () =
  run @@ fun () ->
  Parser.Printconfig.Reader.run ~env:{ style = `Noncompact; state = `Case; chars = `Unicode }
  @@ fun () ->
  reformat "f x ` hello\n` goodbye\n y z";
  reformat "f x {` hello `}\n` goodbye\n y z";
  reformat "f x {` hello\nworld `}\n` goodbye\n y z";
  reformat "f x \n` hello\n y z";
  reformat "f x\n y z";
  reformat "f x\n\n y z"
