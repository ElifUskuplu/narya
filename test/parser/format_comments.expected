f x ` hello
    ` goodbye
  y z

f x {` hello `}
    ` goodbye
  y z

f x {` hello
world `}
    ` goodbye
  y z

f x
  ` hello
  y z

f x y z

f x
  
  y z
